packages/video_player/video_player_android/android/src/main/java/io/flutter/plugins/videoplayer/VideoPlayer.java

```diff
@@ -214,6 +230,9 @@ public void onPlayerError(final PlaybackException error) {
             }
           }
         });
+
+    this.exoPlayer = exoPlayer;
+    this.eventSink = eventSink;
```

Comment:
Should we do these at the start of the method, in case `setUpVideoPlayer` ever calls a method that uses the members?

Reply:
That makes sense to me, but the reason I didn't do that is because I would have to specify the `this` for each usage of these in this method. If that's not a problem, then I'll change it!

Reply:
Why would we need to change the other uses? Wouldn't that only be true if we were assigning new values to the arguments (which is generally not a good idea anyway from a readability perspective)?

Reply:
Oh, I see. That was a misunderstanding on my part.